APIExamples/README.md

```diff
@@ -0,0 +1,27 @@
+## Cortex OpenAPI Postman examples
+
+The files contained in this folder are intended for use with Postman to assist users in familiarizing themselves with the Cortex Fabric OpenAPIs.
+
+## How to use API Examples
+
+1. Import both files to your Postman instance.
+
+2. You must set the environment URL to your DCI instance and [your JWT token](/getting-started/access.md) in the example environment file.
```

Comment:
`[your JWT token](/getting-started/access.md)` --> this needs to be a full link to the outside cortex-fabric site: https://cognitivescale.github.io/cortex-fabric/docs/getting-started/access/